src/main/java/Student.java

```diff
@@ -0,0 +1,25 @@
+public class Student {
+    private String matricNumber;
+    private double GPA;
+
+    public Student(String matricNumber) {
+        this.matricNumber = matricNumber;
+        this.GPA = 0.0;
+    }
+
+    public String getMatricNumber() {
+        return matricNumber;
+    }
+
+    public double getGPA() {
+        return GPA;
+    }
+
+    public void setMatricNumber(String matricNumber) {
+        this.matricNumber = matricNumber;
+    }
```

Comment:
Do we need a setter for this since it is unlikely a student will be able to change their matricNumber?